### ISSUE TITLE
Incorrect LaTeX rendering

### ISSUE THREAD

**Issue by trevorcampbell** [jupyterlab-contrib/rise#75]
## Issue

LaTeX "hat" accents are not rendered correctly. I'm not sure, but it may be related to #72 .

[Screencast from 2024-03-05 11:01:39 AM.webm](https://github.com/jupyterlab-contrib/rise/assets/5981307/5f5cbca0-c727-4c60-ae74-f609d6495a6d)

## Reproduce

Open a slides notebook. Change a cell to markdown. Type `$\hat{y}$` into the cell, and view in presentation mode.

## Expected behavior

I would expect LaTeX to be rendered correctly.

## Context

Using Chrome Version 122.0.6261.69 (Official Build) (64-bit).

Output of `pip list | grep jupyter`
```
jupyter_client                8.4.0
jupyter_core                  5.4.0
jupyter-events                0.8.0
jupyter-lsp                   2.2.0
jupyter-resource-usage        1.0.1
jupyter_server                2.8.0
jupyter-server-mathjax        0.2.6
jupyter_server_terminals      0.4.4
jupyter-telemetry             0.1.0
jupyterhub                    4.0.2
jupyterlab                    4.0.7
jupyterlab_git                0.50.0
jupyterlab-pygments           0.2.2
jupyterlab_rise               0.42.0
jupyterlab_server             2.25.0
jupyterlab-widgets            3.0.9

```

Output of `jupyter labextension list`
```
JupyterLab v4.0.7
/opt/conda/share/jupyter/labextensions
        jupyterlab_pygments v0.2.2 enabled  X (python, jupyterlab_pygments)
        nbgrader v0.9.1 enabled OK (python, nbgrader)
        jupyterlab-rise v0.42.0 enabled OK (python, jupyterlab_rise)
        nbdime-jupyterlab v3.0.1 enabled OK
        @jupyter-notebook/lab-extension v7.0.6 enabled OK
        @jupyter-widgets/jupyterlab-manager v5.0.9 enabled OK (python, jupyterlab_widgets)
        @jupyterlab/git v0.50.0 enabled OK (python, jupyterlab-git)
        @jupyter-server/resource-usage v1.0.1 enabled OK (python, jupyter-resource-usage)


   The following extensions are outdated:
        jupyterlab_pygments
        
   Consider checking if an update is available for these packages.


Disabled extensions:
    @jupyterlab/cell-toolbar-extension (all plugins)
```

**Comment by briank-git** [comment]
I have the same issue, here is the error output from JupyterLab:

```
[W 2024-08-08 11:02:37.178 ServerApp] 403 GET /static/rise/1cb1c39ea642f26a4dfe.woff (127.0.0.1): 1cb1c39ea642f26a4dfe.woff is not in root static directory
[W 2024-08-08 11:02:37.180 ServerApp] 403 GET /static/rise/1cb1c39ea642f26a4dfe.woff (5685f6367514464e8102bf5f9fd79430@127.0.0.1) 1.61ms referer=http://127.0.0.1:8888/rise/examples/issue-75.ipynb?activeCellIndex=0
[W 2024-08-08 11:02:37.199 ServerApp] 403 GET /static/rise/a009bea404f7a500ded4.woff (127.0.0.1): a009bea404f7a500ded4.woff is not in root static directory
[W 2024-08-08 11:02:37.201 ServerApp] 403 GET /static/rise/a009bea404f7a500ded4.woff (5685f6367514464e8102bf5f9fd79430@127.0.0.1) 2.61ms referer=http://127.0.0.1:8888/rise/examples/issue-75.ipynb?activeCellIndex=0
```

It seems either MathJax can't find the font file or it's in the wrong place.

**Comment by DrIdle** [comment]
I had the same problem, but after installing `jupyterlab-mathjax3` the LaTeX output was rendered correctly. Although I can still see the 403 warnings about the `.woff' files in the terminal output.

**Comment by briank-git** [comment]
Can also confirm that installing `jupyterlab-mathjax3` fixed the issue. I put in a PR to add it as a dependency.

**Comment by fcollonval** [comment]
Closing as fixed by #102
